src/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,22 +14,17 @@ const ProgressBar = ({ step }: ProgressBarProps) => {
   const { theme, formData } = useContext(Context);
   const totalSteps = getStepDirectory(formData.immutableReferrer).length + STARTING_QUESTION_NUMBER;
   const { id } = useParams();
-  let stepValue: number = 0;
-  if (step !== undefined) {
-    stepValue = step;
-  } else if (id !== undefined) {
-    stepValue = Number(id);
-  }
+
+  let stepValue = step ?? id ?? 0;
+  let progressPercentage: number = ((Number(stepValue) - 1) / totalSteps) * 100;
 
   const progressBarStyles = {
     marginBottom: '5px',
-    backgroundColor: '#d6d6d6c4',
-    border: '1px solid #B0B0B0',
+    backgroundColor: theme.secondaryBackgroundColor,
```

Comment:
Are we changing the background on the progress bar? If not, it is okay to leave the hard coded grey value.

Reply:
Ok. I've reverted the grey value.